Source/Core/DolphinQt2/Config/Graphics/GraphicsChoice.cpp

```diff
@@ -14,12 +15,14 @@ GraphicsChoice::GraphicsChoice(const QStringList& options, const Config::ConfigI
           &GraphicsChoice::Update);
   setCurrentIndex(Config::Get(m_setting));
 
-  if (Config::GetActiveLayerForConfig(m_setting) != Config::LayerType::Base)
-  {
-    QFont bf = font();
-    bf.setBold(true);
-    setFont(bf);
-  }
+  connect(&Settings::Instance(), &Settings::EmulationStateChanged, [this]() {
```

Comment:
Why does this need to update on EmulationStateChanged? I don't see anything that depends on the emulation state. Seems like using AddConfigChangedCallback instead is what we want.